content/games/factions/companies.md

```diff
@@ -73,6 +80,12 @@ Vicious (3)
 Well-Armed (1)
 : Once each turn, may reroll all results of 1 on a single to-hit roll.
 
+| Taunt                 | Check | Requirements                                                                     | Effect                                                                                                                                                                                                                                           |
+| :-------------------- | :---: | :------------------------------------------------------------------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Blood Makes Mud!      |  6+   | Shieldkin only. Must be within 6" of a melee.                                    | When this Group Attacks an enemy already in melee, they roll three extra dice. While in melee, this Group is Unfeeling.                                                                                                                          |
+| Our Bolts, Your Eyes! |  7+   | Mounted Crossbows only. Must be within 12" of an enemy.                          | This Group becomes Reckless, except they must Shoot instead of Attack if they are able. They only hit on 6 when Shooting and each hit counts twice.                                                                                              |
+| To Glory or Hell!     |  8+   | Lancers only. Must be within 10" of an enemy Captain or Group worth more points. | This group gains the Self-Destruct Trait. If they fail when testing Resolve, they self-destruct instead of retreating. If they self-destruct, roll 2d6 for each remaining Group they commanded; on results of 6+, those Groups gain this effect. |
```

Comment:
```suggestion
| To Glory or Hell!     |  8+   | Lancers only. Must be within 10" of an enemy Captain or another enemy Group worth more points than this one. | This group gains the Self-Destruct Trait. If they fail when testing Resolve, they self-destruct instead of retreating. If they self-destruct, roll 2d6 for each remaining Group they commanded; on 6+, those Groups gain this effect. |
```

Reply:
im probably just spacing on this one, but have we established what "commanded" means in this context? im not sure what this means atm

Reply:
Oh, shit, I don't think we have. I think the correct interpretation here is "remaining Group in their Company."

Reply:
Cool, updated the suggestion above to reflect.